app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapControls.jsx

```diff
@@ -160,12 +161,13 @@ export default class SamplesHeatmapControls extends React.Component {
   };
 
   renderThresholdFilterSelect() {
+    const { options, selectedOptions } = this.props;
     return (
       <ThresholdFilterDropdown
-        options={this.props.options.thresholdFilters}
-        thresholds={this.props.selectedOptions.thresholdFilters}
+        options={options.thresholdFilters}
+        thresholds={selectedOptions.thresholdFilters}
         onApply={this.onThresholdFilterApply}
-        disabled={this.props.loading || !this.props.data}
+        hideBadgeIfInsufficientSpace={true}
```

Comment:
Nit: You can just do `hideBadgeIfInsufficientSpace` instead of `hideBadgeIfInsufficientSpace={true}`